src/main/java/com/cloudbees/jenkins/support/BundleBrowser.java

```diff
@@ -0,0 +1,108 @@
+package com.cloudbees.jenkins.support;
```

Comment:
@christ66 I've checked the GSoC rules, and there is no explicit guideline about it. Seems you can go forward with this naming